bears/js/JSHintBear.py

```diff
@@ -16,19 +31,377 @@ class JSHintBear(LocalBear, Lint):
+        :param environment_phantom:             This option defines globals
+                                                available when your core is
+                                                running inside of the PhantomJS
+                                                runtime environment.
+        :param maxerr:                          This options allows you to set
+                                                the maximum amount of warnings
+                                                JSHintBear will produce before
+                                                giving up. Default is 50.
+        :param maxstatements:                   Maximum number of statements
+                                                allowed per function.
+        :param maxdepth:                        This option lets you control how
+                                                nested do you want your blocks
+                                                to be.
+        :param maxparams:                       Maximum number of formal
+                                                parameters allowed per function
+        :param maxcomplexity:                   Maximum cyclomatic complexity in
+                                                the code.
+        :param shadow:                          This option suppresses warnings
+                                                about variable shadowing i.e.
+                                                declaring a variable that had
+                                                been already declared somewhere
+                                                in the outer scope.
+                                                 - "inner" - check for variables
+                                                             defined in the same
+                                                             scope only
+                                                 - "outer" - check for variables
+                                                             defined in outer
+                                                             scopes as well
+                                                 - false   - same as inner
+                                                 - true    - allow variable
+                                                             shadowing
+        :param prohibit_unused:                 This option genrates warnigns
+                                                when variables are defined but
+                                                never used. This can be set to
+                                                ``vars`` to only check for
+                                                variables, not function
+                                                parameters,or ``strict`` to
+                                                check all variables and
+                                                parameters.
```

Comment:
If `vars` and `strict` are strings, enclose them inside quotes please, so no one thinks these are special values :+1: